components/integrations/bots/bot.test.tsx

```diff
@@ -3,30 +3,36 @@
 
 import React from 'react';
 import {shallow} from 'enzyme';
-
 import {FormattedMessage} from 'react-intl';
 
 import Markdown from 'components/markdown';
-
 import TestHelper from 'tests/helpers/client-test-helper';
+import {TestHelper as UtilsTestHelper} from 'utils/test_helper';
 
-import Bot from './bot.jsx';
+import Bot from './bot';
 
 describe('components/integrations/bots/Bot', () => {
-    const team = TestHelper.fakeTeam();
+    const team = UtilsTestHelper.getTeamMock();
```

Comment:
This fakeTeam function doesn't generate an object that matches all the required properties of the Team type so I used this other method instead. The same applies to some other mocked objects here.